.build/build-resolver.xml

```diff
@@ -246,6 +247,15 @@
             <file file="${local.repository}/org/apache/cassandra/deps/sigar-bin/libsigar-x86-linux.so"/>
             <file file="${local.repository}/org/apache/cassandra/deps/sigar-bin/libsigar-x86-solaris.so"/>
         </copy>
+
+        <copy todir="${build.lib}/corretto" quiet="true">
+            <file file="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"/>
```

Comment:
don't like the hard coded 2.2.0 -- can't there be vars like in maven?

Reply:
@xgerman There is a lot of instances this is happening in that file, if you look closed. If we fixed it just here, it would be inconsistent with everything else. I would rather dedicated a separate ticket for cleaning it all up in one shot. How does that sound to you?

Reply:
Sounds good to me.